builder/amazon/common/access_config_test.go

```diff
@@ -40,7 +30,7 @@ func TestAccessConfigPrepare_Region(t *testing.T) {
 }
 
 func TestAccessConfigPrepare_RegionRestricted(t *testing.T) {
-	c := testAccessConfig()
+	c := FakeAccessConfig()
```

Comment:
To make this exportable, I had to change the name or else golang would have thought it was a test